benchmark/benchmark-sysbench-v4-vs-v3.md

```diff
@@ -0,0 +1,174 @@
+---
+title: TiDB Sysbench Performance Test Report -- v4.0 vs. v3.0
+summary: Compare the Sysbench performance of TiDB 4.0 and TiDB 3.0.
+category: benchmark
+---
+
+# TiDB Sysbench Performance Test Report -- v4.0 vs. v3.0
+
+## Test purpose
+
+This test aims to compare the Sysbench performances of TiDB 4.0 and TiDB 3.0 in the OLTP (Online Transactional Processing) scenario.
+
```

Comment:
Just like [other benchmark](https://pingcap.com/docs/stable/benchmark/v3.0-performance-benchmarking-with-tpcc/) reports, we should point put "TiDB version: v3.0.13 vs. v4.0.0." 
@Little-Wallace